docs/usage/quickstart.rst

```diff
@@ -67,6 +67,8 @@ To create an inventory:
    default configuration) on hosts in the *kube-node* group, that is,
    hosts on which Pods will be scheduled::
```

Comment:
In a `.. code-block:: yaml` (or similar) section the `::` is no longer required, can be plain `:`.